### DISCUSSION TITLE
Funktioniert der HmIP-DRSI4 mit hap-homematic ?

### DISCUSSION THREAD

**Discussion by icefish-creativ** [thkl/hap-homematic#635] [Q&A]
Hat jemand den Aktor erfolgreich am laufen ?

**Comment by andreas0205** [comment]
Hallo Icefish, Ich habe das bisher auch erfolglos versucht. Hast du mittlerweile eine Lösung dafür?

**Reply by icefish-creativ** [reply]
Hi Andreas,
leider nein , die Bridge von Mediola Matter Bridge soll laut Doku den DRSI4 interstützen. 

Die mangelde unterstützung für Homekit lässt mich eher über altertnativen zu Homematic nachdenken ....

**Comment by andreas0205** [comment]
Hallo Tim,
Ich habe das Problem gestern gelöst.
1. Füge ein neues Gerät hinzu (Siehe Bild)

2. Wähle unter Service: HomeMaticSwitchAccessory (Wichtig -> Nicht HomeMaticProgrammableSwitchAccessory)
3. Dann wähle noch die HAP Instanz und es sollte gehen.

Ich bin komplett auf HomeMatic, d.h. ich werde jetzt alle meine Geräte einbinden und sollte etwas nicht gehen, ggf. auch in GitHub den Code anpassen ;)

Gruß und lass mich wissen ob es klappt.
Das bekommen wir schon hin.

**Reply by icefish-creativ** [reply]
Hi Andreas,

sehe kein Bild ...

sehr cool. Erscheint der dann als 4 einzelne Schalter im Homekit ?

Gruss Tim

**Comment by andreas0205** [comment]
Ja du musst jeden Schalter einzeln  hinzufügen.Am 01.10.2024 um 19:42 schrieb Tim Zöllner ***@***.***>:﻿
Hi Andreas,
sehe kein Bild ...
sehr cool. Erscheint der dann als 4 einzelne Schalter im Homekit ?
Gruss Tim
